src/app/api/example/route.ts

```diff
@@ -8,4 +8,4 @@ import { NextResponse } from "next/server";
 export async function GET() {
   await connectDB();
   return NextResponse.json({ message: "Hello from the API!" });
-}
+}
```

Comment:
Please don't make changes to unnecessary files. You also edited whitespace in the `eventSchema.ts`